fakebook/src/app/app.module.ts

```diff
@@ -3,10 +3,12 @@ import { NgModule } from '@angular/core';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { NotificationsViewComponent } from './notifications-view/notifications-view.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    NotificationsViewComponent
```

Comment:
this isn't needed for the tests and so it technically doesn't belong in this PR.
however because ng generate adds it i'll let it through for convenience.